### PR TITLE
rootCmd:add choice version action

### PR DESCRIPTION
**Pull request by Theodoree**

### CODE REVIEW

**Comment by Theodoree** [comment]
Add option to switch version, just like node version manager tool

**Comment by owenthereal** [comment]
@Theodoree I like this idea! Would you mind converting the multiple-choice library to https://github.com/manifoldco/promptui/? This is an example of how to do that: https://github.com/manifoldco/promptui/blob/master/_examples/select/main.go, and I introduced this library to master: https://github.com/owenthereal/goup/pull/14

**Comment by Theodoree** [comment]
@owenthereal  done

File: go.mod [hunk 1]
```diff
@@ -8,4 +8,5 @@ require (
 	github.com/sirupsen/logrus v1.2.0
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5 // indirect
+	golang.org/x/sys v0.0.0-20200909081042-eff7692f9009 // indirect
```
Comment (owenthereal) [inline go.mod]: Can you remove this dependency? You can run a `go mod tidy` and `go mod vendor` to clean it up.

File: internal/commands/root.go [hunk 2]
```diff
@@ -73,3 +74,47 @@ func GoupDir(paths ...string) string {
...
+		return err
+	}
+
+	if len(vers) == 0 {
+		showGoIfExist()
+		return nil
+	}
+
+	var curVer string
+	var pos int
```
Comment (owenthereal) [inline internal/commands/root.go]: Minor, you can wrap vars in a group like this:

```go
var (
  curVar string
  pos int
)
```

File: internal/commands/root.go [hunk 3]
```diff
@@ -73,3 +74,47 @@ func GoupDir(paths ...string) string {
...
+	for idx, v := range vers {
+		items = append(items, v.Ver)
+		if v.Current {
+			curVer = v.Ver
+			pos = idx
+		}
+	}
+
+	prompt := promptui.Select{
+		Label:     "select a version",
```
Comment (owenthereal) [inline internal/commands/root.go]: It would read better if you capitalize as "Select a version".

File: internal/commands/root.go [hunk 4]
```diff
@@ -35,6 +35,7 @@ func NewCommand() *cobra.Command {
 	rootCmd := &cobra.Command{
 		Use:   "goup",
 		Short: "The Go installer",
+		RunE:  choiceVersion,
```
Comment (owenthereal) [inline internal/commands/root.go]: It reads better if the function is a verb, for example, `runChooseVersion`.

**Review by owenthereal — COMMENTED** [review]
Thanks! Provided some feedback

**Comment by Theodoree** [comment]
@owenthereal  done.

**Comment by owenthereal** [comment]
@Theodoree This is awesome. Thanks for your contribution! I merged your changes in https://github.com/owenthereal/goup/commits/master after a rebase. Closing this PR.

**Pull request closed without merging**